util/filter_bench.cc

```diff
@@ -444,7 +589,8 @@ double FilterBench::RandomQueryTest(bool inside, bool dry_run, TestMode mode) {
     std::cout << "ns/op: " << ns << std::endl;
   }
 
-  if (!inside && !dry_run && mode == kRandomFilter) {
+  if (!dry_run) {
+    fp_rate_report_ = std::ostringstream();
```

Comment:
It may not compatible in gcc 4.9. How about `fp_rate_report_.clear(); fp_rate_report_.str("");` ?

Reply:
If it's not working for you, please create a pull request. You can assign it to me. Thanks

Reply:
https://github.com/facebook/rocksdb/pull/6043 pr as above, plz make a review.